utils/config.py

```diff
@@ -3,6 +3,25 @@
 import os
 import json
 
+from core.config import load_bot_config_from_file, generate_bot_config
+
+try:
+    base_path = (Path(os.path.abspath(__file__)) / '..' / '..').resolve()
+    config_path = base_path / 'config_data.json'
+    config_data = load_bot_config_from_file(config_path.__str__()).get_store_json()
+except FileNotFoundError as fe:
+    print(fe)
+    print("BotConfig error: config_data.json file not found, generating new instance")
+    config_data = generate_bot_config({
+            "youtube_announcement_channels": ["announcements"],
+            "vod_count": 0,  # If the file does not exists evaluate getting this value from a DB
```

Comment:
I think this config file is planned to be the authoritative configuration source for now. (And anyhow, we might change it to use something more reliable than just the count.)